packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -374,6 +376,61 @@ export const components: ThemeOptions["components"] = {
       }),
     },
   },
+  MuiChip: {
+    defaultProps: {
+      deleteIcon: <CloseCircleFilledIcon />,
+    },
+    styleOverrides: {
+      root: ({ theme, ownerState }) => ({
+        height: "auto",
+        paddingBlock: theme.spacing(1),
+        paddingInline: theme.spacing(2),
+        fontSize: theme.typography.body1.fontSize,
+        borderRadius: theme.mixins.borderRadius,
+        backgroundColor: theme.palette.grey[100],
+
+        [`& .${chipClasses.deleteIcon}`]: {
+          WebkitTapHighlightColor: "transparent",
+          color: theme.palette.text.secondary,
+          fontSize: "1em",
```

Comment:
Couldn't we use `inherit` instead of `1em`?

Reply:
They should result in the same presentation. I'm using `1em` here so that this unit stays relative even if someone themes the parent value to be fixed.

For ex, I'd prefer the computed CSS be `1em` rather than `14px`.